Android.gitignore

```diff
@@ -34,3 +34,7 @@ captures/
 
 # Intellij
 *.iml
+.idea
```

Comment:
I'm also hesitant to add this specific rule as [there's a lot of detail](https://github.com/github/gitignore/blob/master/Global/JetBrains.gitignore) to the IntelliJ-based IDEs and what they exclude - see https://github.com/github/gitignore/pull/1475 for the discussion about this - and the links to the documentation